benjaminkott/bootstrap_package
# [BUGFIX] Use contextual icon color for system information icon

# Pull Request

## Prerequisites

* [x] Changes have been tested on PHP 7.4
* [x] Changes have been tested on PHP 8.0
* [x] Changes have been tested on PHP 8.2
* [ ] CSS has been rebuilt (only if there are SCSS changes `cd Build; npm ci; npm run build`)

## Description

With TYPO3 v12 and Bootstrap v5, light and dark mode are partially available within TYPO3 backend. When using light mode, the bootstrap icon in system information toolbar gets hidden, because it's filled with white (`#fff`) and is therefore no longer visible. This can be handled by avoiding a custom `fill`, but instead using the `icon-color` class handled by TYPO3 core.

This solution is backwards compatible to TYPO3 v11.5.

Light mode:

![Bildschirmfoto 2023-06-13 um 10 00 01](https://github.com/benjaminkott/bootstrap_package/assets/16313625/c826c178-ca01-49df-9c9a-7b3b63325b49)

Dark mode:

![Bildschirmfoto 2023-06-13 um 09 59 51](https://github.com/benjaminkott/bootstrap_package/assets/16313625/2d77b256-653e-4d3a-bccc-6e8c16032ab1)

## Steps to Validate

1. Switch to light mode
2. Open TYPO3 backend
3. Open system information toolbar
4. Check visibility of bootstrap package icon
